tests/fixtures/tables/table_source_geog.sql

```diff
@@ -0,0 +1,46 @@
+DROP TABLE IF EXISTS table_source_geog;
+CREATE TABLE table_source_geog(gid serial PRIMARY KEY, geog geography(GEOMETRY, 4326));
+
+INSERT INTO table_source_geog(geog) values ('SRID=4326;POINT(0 0)'::geography);
```

Comment:
we usually prefer to have the same type of elements in a table - if you mix points and linestrings, what would be the type of the MVT data?  Not a blocker, but for a test we may want a few more tables like this .... although perhaps one mixed table is also good to test the results

Reply:
I copied this from the `table_source` test case, which is doing the same for geometries.
I agree that we can probably use a few more tests for the geography columns.
Any recommendations on the best approach for this? I could just copy/paste all the tests we have for geometries to run them on geographies as well, but that feels like a lot of duplication, so I was wondering if there could be a more clever way?